include/ceed/ceed.h

```diff
@@ -216,6 +216,11 @@ CEED_EXTERN const CeedVector CEED_VECTOR_NONE;
 /// @ingroup CeedBasis
 CEED_EXTERN const CeedBasis CEED_BASIS_NONE;
 
+/// DEPRECATED Argument for CeedOperatorSetField that no basis operation is needed to translate between the E-vector and the Q-vector.
+/// Only use this option with CeedEvalMode CEED_EVAL_NONE.
+/// @ingroup CeedBasis
```

Comment:
Not sure how to best demarcate this in the docs, or if we want to have it documented at all.